# DP: mips still has not INLINE_SYSCALL so define __syscall_exit
# DP: status: applied to CVS

--- libc/sysdeps/unix/sysv/linux/mips/syscalls.list.orig	2002-09-22 13:25:21.000000000 +0000
+++ libc/sysdeps/unix/sysv/linux/mips/syscalls.list		2002-09-22 13:25:21.000000000 +0000
@@ -74,6 +74,7 @@
 s_sigprocmask	sigprocmask sigprocmask	i:ipp	__syscall_sigprocmask
 s_stat64	xstat64  stat64		i:sp	__syscall_stat64
 s_truncate64	truncate64 truncate64	i:siii	__syscall_truncate64
+s_exit		_exit	exit		i:i	__syscall_exit
 
 # Todo: we can pass 6 args in registers, no need for the wrapper
 sys_sysctl	sysctl	_sysctl		i:p	__syscall__sysctl
